addons/phone_validation/tests/test_phonenumbers_patch.py

```diff
@@ -19,6 +19,7 @@ def test_region_CI_monkey_patch(self):
         # MONKEY PATCHING phonemetadata of Ivory Coast if phonenumbers is too old
         if parse_version('7.6.1') <= parse_version(phonenumbers.__version__) < parse_version('8.12.32'):
             # check that _local_load_region is set to `odoo.addons.phone_validation.lib.phonenumbers_patch._local_load_region`
+            self.assertEqual(phonenumbers.PhoneMetadata._region_available['CI'], phonenumbers_patch._local_load_region, "The phonenumbers module should have been patched between versions 7.6.1 and 8.12.32")
```

Comment:
i prefer for myself to have long line (so it's easier to grep), but I think generally we try to stay below 100 characters if possible, so this assert and the other ones could be split over several lines, eg.:

```suggestion
            self.assertEqual(
                phonenumbers.PhoneMetadata._region_available['CI'],
                phonenumbers_patch._local_load_region,
                "The phonenumbers module should have been patched between versions 7.6.1 and 8.12.32")
            )
```
since the original files already has 135 character lines I guess we could continue to break the style anyway, so not too bad to keep it as it is currently

Reply:
I also bumped into the situation where I couldn't locate some code, because it was split in multi-lines. I'd prefer to leave it this way, all the code editors I know can deal with that with some line wrap, so its not a hassle

Reply:
On a second thought It's just a test's error message, it's not really something somebody would look with regex search for. Splitting